nemoguardrails/library/zscaler_aiguard/actions.py

```
@@ -0,0 +1,234 @@
+                policy_name,
+            )
+
+        return {
+            "action": action_val,
+            "severity": severity,
+            "policy_name": policy_name,
+            "transaction_id": transaction_id,
+            "detectors": detectors,
+            "blocking_detectors": blocking_detectors,
+            "message": message,
+        }
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)